tests/Fixtures/Integration/set/@PER-CS2.0.test-out.php

```diff
@@ -56,3 +56,18 @@ class Aaa implements
     'foo' => 'bar',
     'foo2' => 'bar',
 ];
+
+$arrayMultilineWithoutComma = [
+    'foo' => 'bar',
+    'foo2' => 'bar',
+];
+$heredocMultilineWithoutComma = [
+    'foo',
+    <<<EOD
+        bar
+        EOD,
+];
+argumentsMultilineWithoutComma(
+    1,
+    2,
```

Comment:
I'm not sure about `arguments` config belonging to PER-CS2.
It's also not listed as example in https://www.php-fig.org/per/coding-style/#26-trailing-commas .
Can you raise a PR to above examples to include the function call arguments ?

I'm all in for that style personally, but I'm not sure it's indeed part of PER-CS. Same time, i know big communities against it (eg Symfony)

Reply:
(originally concern raised in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7916#discussion_r1546759323 )

Reply:
> It's also not listed as example in https://www.php-fig.org/per/coding-style/#26-trailing-commas .

It's listed in the first paragraph though:
> Examples include array key/value pairs, **function arguments**, closure use statements, match() statement branches, etc.

Again, IMO the following covers any existing and future list that allows trailing comma after last item:

> Numerous PHP constructs allow a sequence of values to be separated by a comma, and the final item may have an optional comma.
> […]
> If the list is split across multiple lines, then the last item MUST have a trailing comma.

So I would include function arguments even if not listed explicitly.

Reply:
We had lengthy discussion on `function arguments` on Sf repo - does it apply to function definition `function foo($arg) {...}` , call `foo(1)` or both.

I recognise that some communities are against that rule and claiming that PER is not conclusive on it - thus idea to enrich the examples in PER docs ;)

but anyway, as I said I'm happy for that style myself, I'm happy we do not enable it for Sf ruleset.

if we enable it by default to PER ruleset - in case ppl would complain (i hope not), let's drop that one config item from ruleset and push to extend PER docs (and bring it back to ruleset)